lib/cmd/common/cluster.js

```diff
@@ -85,11 +95,9 @@ function userUpdate(args, cb) {
 
   var params = {};
   for (var i = 2; i < args.length; i++) {
-    if (args[i]) {
-      if (args[i].indexOf("=") !==-1) {
-        var kv = args[i].split("=");
-        params[kv[0]] = kv[1];
-      }
+    if (args[i] && args[i].indexOf("=") !== -1) {
```

Comment:
This is just stylistic, and other usages of `indexOf` compare with `-1` in some way or another, so I think we should err on the side of less changes.